package.json

```diff
@@ -87,17 +93,17 @@
     "server/sonarlint-ls.jar"
   ],
   "scripts": {
-    "vscode:prepublish": "tsc -p ./ && node scripts/prepare.js",
-    "compile": "tsc -watch -p ./",
+    "vscode:prepublish": "./node_modules/.bin/tsc -p ./ && node scripts/prepare.js",
+    "compile": "./node_modules/.bin/tsc -watch -p ./",
```

Comment:
@henryju do we need this `watch`?

Reply:
No idea. It certainly come from the original template we used as a quick starter. Do you suggest we should remove it?

Reply:
@henryju in fact it does nothing:) it should be `-w` or `--watch`. So you can remove it.